src/include/gdb_packet.h

```diff
@@ -25,6 +25,8 @@
 #include <stdarg.h>
 #include <stdbool.h>
 
+#include "debug.h"
```

Comment:
This header is BMDA-only - we should probably put a clone of the printf-like format string logic into the general header to work around this, but customised to take the two parameter numbers explicitly so it can be applied later to things like tc_printf().